packages/api-client/src/types/GraphQL.ts

```diff
@@ -160,6 +162,13 @@ export type Collection = Node & {
   customFields?: Maybe<Scalars['JSON']>;
 };
 
+export type ProductCollection = {
```

Comment:
I don't remember this kind of type. Have you added it manually? If so, I would recommend moving this type into other type files in the project because if you add a type here and then we will generate typings once more this type won't be applied. Use /types/types.ts file instead.

Reply:
I have added it manually. I'll move it to `/types/types.ts`